data/orders.json

```diff
@@ -1,505 +1,711 @@
-[{
-  "id": "abc",
-  "createdAt": "1970-01-01T00:00:00.001Z",
-  "lastModifiedAt": "1970-01-01T00:00:00.002Z",
-  "orderNumber": "10001",
-  "version": 1,
-  "lineItems": [{
-    "id": "LineItemId-1-1",
-    "productId": "ProductId-1-1",
-    "name": {
-      "en": "Product-1-1"
-    },
-    "variant": {
-      "prices": [
-        {
-          "value": {
-            "type": "centPrecision",
-            "currencyCode": "EUR",
-            "centAmount": 12900,
-            "fractionDigits": 2
-          },
-          "id": "5e7734cf-59ca-42d4-bb21-b8c01d0db58d",
-          "country": "DE"
+[
```

Comment:
Could we please revert the formatting of this file? It's really difficult to see your actual changes 😕 Or at the very least, could you please point out the changes?

Reply:
A new order was added to the test data ([see here](https://github.com/sphereio/sphere-order-export/blob/25b929cca1fc7e5de150cca586bacda5aa8bec17/data/orders.json#L548)).